legal-api/src/legal_api/models/party_member.py

```diff
@@ -0,0 +1,70 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module holds data for people related to a party role for a business."""
+from .db import db
+
+
+from .address import Address  # noqa: F401 pylint: disable=unused-import; needed by the SQLAlchemy relationship
+
+
+class PartyMember(db.Model):  # pylint: disable=too-many-instance-attributes
```

Comment:
Draw values, names and lengths from https://schema.org/Person
verify lengths against Colin and see what transforms are needed.

Reply:
I changed the lengths for the fields that exist in colin to match. Other than that I kept it the same as the director model